packaging/rhel/xrootd.spec.in

```diff
@@ -653,7 +653,15 @@ popd
 pushd xrootd
 pushd  build
 make install DESTDIR=$RPM_BUILD_ROOT
-cat PYTHON_INSTALLED | sed -e "s|$RPM_BUILD_ROOT||g" > PYTHON_INSTALLED_FILES
+
+%if %{python3only}
```

Comment:
I'm not very familiar with the RPM packaging spec, so I did this as there [doesn't seem to be a conditional](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/rpm_packaging_guide/index#rpm-conditionals-syntax_rpm-conditionals) for something like `%ifnot %{python3only}`. If there is negation, then this can be simplified.

Reply:
Negation with `%if !%{python3only}` seems to work.